File: leetcode/easy/richest-customer-wealth.cpp
// https://leetcode.com/problems/richest-customer-wealth/

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int high = 0;
        for(int i = 0; i < accounts.size(); i++)
        {
            int count = 0;
            for(int j = 0; j < accounts[i].size() ;j++)
            {
                count += accounts[i][j];
            }
            high = max(high, count);
        }
        return high;
    }
};

File: array/richest_customer_wealth.go
package array

func maximumWealth(accounts [][]int) int {
	var res int = 0
	for i := 0; i < len(accounts); i++ {
		var sum int
		for j := 0; j < len(accounts[i]); j++ {
			sum = sum + accounts[i][j]
		}
		if sum > res {
			res = sum
		}
	}
	return res
}
